src/main/java/com/terraformersmc/terrestria/init/TerrestriaFeatures.java

```diff
@@ -142,16 +145,7 @@ public static void init() {
 
 		VOLCANO_PIECE = Registry.register(Registry.STRUCTURE_PIECE, new Identifier(Terrestria.MOD_ID, "volcano"), VolcanoGenerator::new);
 
-		// TODO
-		/*SAKURA_LEAF_PILE_DECORATOR = Registry.register(
-				Registry.TREE_DECORATOR_TYPE, new Identifier(Terrestria.MOD_ID, "sakura_leaf_pile"),
-				MixinTreeDecoratorType.create(SakuraLeafPileDecorator::new)
-		);
-
-		FIX_SMALL_LOGS_DECORATOR = Registry.register(
-				Registry.TREE_DECORATOR_TYPE, new Identifier(Terrestria.MOD_ID, "fix_small_logs"),
-				MixinTreeDecoratorType.create(FixSmallLogsDecorator::new)
-		);*/
```

Comment:
I was waiting on the mixin constructor issue to be resolved before adding these registrations back